build.gradle.kts

```diff
@@ -4,7 +4,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 val githubRepo = "jakobkmar/KSpigot"
 
 group = "net.axay"
-version = "1.20.4"
+version = "1.20.5"
```

Comment:
shouldn't the version be 1.21, not 1.20.5?

Reply:
The kspigot Version is not equal to the mc version

Reply:
yeah but the latest major minecraft version bumps also bumped the kspigot version up

Reply:
Yeah, but there is No Major Change in kspigot

Reply:
That's true but it would create a lot of confusion for developers when the major kspigot version no longer matches the Minecraft version, so I think bumping the kspigot version to 1.21 would make the most sense.

Reply:
I See that, idk i think its Jakobs decision. So we'll see.

Reply:
But I still think that kspigot doesn't have to have the same version as paper. It's a standalone project. I don't see any reason to deviate from the normal version pattern.

Reply:
The normal version pattern was to bump the major version of kspigot with the major version of minecraft, see the [1.20.1](https://github.com/jakobkmar/KSpigot/releases/tag/1.20.1) release, there were no major changes either. Either way I think this discussion is stupid and @jakobkmar should just decide it, when he sees this.

Reply:
@xNoobyyy correct, it is "major minecraft version _dot_ kspigot release number" - a bit confusing but not worth changing it